client/src/components/NavBar/NotificationRequest/NotificationRequest.tsx

```diff
@@ -24,11 +24,11 @@ const NotificationRequest: React.FC<IProps> = ({
 
   return (
     <Box display="flex" alignItems="center" m={2}>
-      <Avatar variant="square" src={`/${image}`} className={classes.large} />
+      <Avatar variant="square" src={`/${senderProfile.image}`} className={classes.large} />
```

Comment:
Since this should be a URL instead of a relative path:
```suggestion
      <Avatar variant="square" src={senderProfile.image} className={classes.large} />
```